addons/strings/fnc_capitalize.sqf

```diff
@@ -25,16 +25,19 @@ Author:
 ---------------------------------------------------------------------------- */
 SCRIPT(capitalize);
 
-params ["_string"];
+forceUnicode 0;
```

Comment:
side effect?

Reply:
The function should work properly now.

The old code would convert the first byte only, not guaranteeing if it was a char or just part of one (ANSI vs Unicode).

I understand if this change isn't wanted for backward compatibility reasons.

Reply:
What I mean is, calling this function has the side effect of changing the unicode mode of the script it was called from.

Reply:
I should have looked at the wiki example a bit more in detail. I'll address that problem asap.

Reply:
I found a workaround. Now, calling `CBA_fnc_capitalize` does not affect the parent scope's unicode support at all.